alodataset/coco_base_dataset.py

```diff
@@ -13,7 +13,7 @@
 
 from alodataset import BaseDataset
 from aloscene import BoundingBoxes2D, Frame, Labels, Mask
-
+from pathlib import Path
```

Comment:
can you alphabetically sort this files imports ?

Reply:
I actually commited this file by mistake, it was already merged in #247